ui/imports/shared/views/PasswordView.qml

```diff
@@ -244,7 +245,7 @@ Column {
             id: strengthInditactor
             width: parent.width
             anchors.horizontalCenter: parent.horizontalCenter
-            value: newPswInput.text.length > root.minPswLen ? root.minPswLen : newPswInput.text.length
+            value: Math.min(root.minPswLen, newPswInput.text.length ? Math.max(root.showPswLen, newPswInput.text.length) : 0)
```

Comment:
I'm not sure about that fix.. The bar value needs to change progressibly while you enter chars, and with the first character it increases, let's say, to the second stage.. Did you agree this behavior with the designer? 
My suggestion should be more to truncate and write dots if the text doesn't fit in the bar value like here happens with the "Very weak" text. If we go with my suggestion, then it should be addressed in `StatusQ`. 
Could you review it with the designer if not done before, please?

Reply:
Thanks for noticing that, discussion: https://discordapp.com/channels/624205794384281629/730410574428504166/974354691158859776

Reply:
Ready!